test/visual/test.js

```diff
@@ -77,6 +77,18 @@ gemini.suite('vaadin-grid', (rootSuite) => {
       });
     });
 
+    gemini.suite(`disabled-${theme}`, (suite) => {
+      suite
+        .setUrl(`disabled.html?theme=${theme}`)
+        .setCaptureElements('.capture-block')
+        .capture('disabled', {}, (actions, find) => {
```

Comment:
nit: no need to set `disabled` dynamically, you can add it as attribute to the web component.

Reply:
Thanks, done.